site/source/docs/tools_reference/emsdk.rst

```diff
@@ -66,8 +66,8 @@ The current set of available :term:`tools <Tool>` and :term:`SDKs <SDK>` are lis
 
 The :term:`SDK` targets are a convenience mechanism for specifying the full set of tools used by a particular Emscripten release. For example, the two lines below are equivalent: ::
 
-  ./emsdk install sdk-incoming-64bit
-  ./emsdk install git-1.8.3 clang-incoming-64bit node-0.10.17-64bit python-2.7.5.3-64bit java-7.45-64bit emscripten-incoming
+  ./emsdk install sdk-upstream-main-64bit
+  ./emsdk install git-1.8.3 clang-upstream-main-64bit node-0.10.17-64bit python-2.7.5.3-64bit java-7.45-64bit emscripten-upstream-main
```

Comment:
It looks like this line needs two other adjustments to be correct:

`'llvm-git-main-64bit'` -> `llvm-git-main-64bit`
`emscripten-upstream-main` -> `emscripten-main-64bit`